app/src/main/java/com/example/ncov19traking/ui/CovidCustomCard.kt

```diff
@@ -0,0 +1,86 @@
+package com.example.ncov19traking.ui
+
+import android.content.Context
+import android.graphics.Color
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import android.widget.TextView
+import androidx.core.content.ContextCompat
+import com.example.ncov19traking.R
+import com.google.android.material.card.MaterialCardView
+
+const val DEFAULT_TITLE_TEXT = "Title"
```

Comment:
What happens when you need internationalization, what problem will you have here? fix it.